### ISSUE TITLE
Option to render SVGs as images

### ISSUE THREAD

**Issue by sammorley-short** [ForkIssues/Tracker#1262]
Currently SVGs are rendered as text. In some cases this might be useful, but it would also be helpful to provide the option to render SVGs as images, similar to how PNGs are currently displayed.

Thanks!

**Comment by Stanzilla** [comment]
Yes please

**Comment by mstarke** [comment]
The current version (2.51.3, dev update channel) shows svgs as images. This is nice if you want to compare the images visually but my workflow requires to compare them as text. I do not see an option to toggle the diff mode back to the text version.

Is there an option that I'm not aware of or does fork not support the svg-text-diff anymore?

**Comment by DanPristupov** [comment]
No, there is no option for this. And will not be easy to implement, I think.

**Comment by mstarke** [comment]
Bummer. But I guess it's an edge case not worth the effort. The workaround then is to use an external diff tool for the svg text diffs.

Thanks for the quick response! And keep up the good work on this awesome tool!

**Comment by Stanzilla** [comment]
Instead of a toggle, would it be possible to just always display both, the image and the code?
